docs/api.md

```diff
@@ -1694,10 +1698,6 @@ This function also returns a `Promise`, with `void` as its argument upon complet
 
 Move an item from `sourceSlot` to `destSlot` in the current window.
 
-#### bot.updateHeldItem()
```

Comment:
updateHeldItem is still exposed https://github.com/PrismarineJS/mineflayer/blob/master/lib/plugins/inventory.js#L525
it should be removed from the code before than from  the documentation